core/src/main/java/org/apache/spark/shuffle/sort/ShuffleExternalSorter.java

```diff
@@ -123,6 +126,8 @@
     this.inMemSorter = new ShuffleInMemorySorter(
       this, initialSize, conf.getBoolean("spark.shuffle.sort.useRadixSort", true));
     this.peakMemoryUsedBytes = getMemoryUsage();
+    this.diskWriteBufferSize =
+            conf.getInt("spark.shuffle.spill.diskWriteBufferSize", DISK_WRITE_BUFFER_SIZE);
```

Comment:
@cloud-fan 
thanks for review it.
Thank you for your advice. I tried to fix it.
However, org.apache.spark.internal.config cannot be imported in the ShuffleExternalSorter.java class
This modification will change the org.apache.spark.internal.config and affect other code changes, 
so I suggest using the PR to modify it. Do you agree?